boards/arm/ubx_evkannab1_nrf52832/ubx_evkannab1_nrf52832.dts

```diff
@@ -0,0 +1,233 @@
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			/* ANNA-B1 GPIO_29 */
+			gpios = <&gpio0 27 GPIO_ACTIVE_LOW>;
+			label = "Red LED";
+		};
+		led1: led_1 {
+			/* ANNA-B1 GPIO_30 */
+			gpios = <&gpio0 25 GPIO_ACTIVE_LOW>;
+			label = "Green LED";
+		};
+		led2: led_2 {
+			/*ANNA-B1 GPIO_31 */
+			gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 27>;
+		};
+		pwm_led1: pwm_led_1 {
+			pwms = <&pwm0 25>;
+		};
+		pwm_led2: pwm_led_2 {
+			pwms = <&pwm0 26>;
+		};
+		red_pwm_led: red_pwm_led {
+			pwms = <&pwm0 27>;
+		};
+		green_pwm_led: green_pwm_led {
+			pwms = <&pwm0 25>;
+		};
+		blue_pwm_led: blue_pwm_led {
+			pwms = <&pwm0 26>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			/*  EVK-ANNA-B1 SW1 button */
+			gpios = <&gpio0 25 (GPIO_PULL_UP | \
+				GPIO_ACTIVE_LOW)>;
+			label = "Push button switch SW1";
+		};
+		button1: button_1 {
+			/* EVK-ANNA-B1 SW2 button */
+			gpios = <&gpio0 24 (GPIO_PULL_UP | \
+				GPIO_ACTIVE_LOW)>;
+			label = "Push button switch 1";
+		};
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 4 0>,	/* A0 */
+			   <1 0 &gpio0 5 0>,	/* A1 */
+			   <2 0 &gpio0 28 0>,	/* A2 */
+			   <3 0 &gpio0 29 0>,	/* A3 */
+			   <4 0 &gpio0 30 0>,	/* A4 */
+			   <5 0 &gpio0 31 0>,	/* A5 */
+			   <6 0 &gpio0 2 0>,	/* D0 */
+			   <7 0 &gpio0 3 0>,	/* D1 */
+			   <8 0 &gpio0 19 0>,	/* D2 */
+			   <9 0 &gpio0 11 0>,	/* D3 */
+			   <10 0 &gpio0 27 0>,	/* D4 */
+			   <11 0 &gpio0 26 0>,	/* D5 */
+			   <12 0 &gpio0 10 0>,	/* D6 */
+			   <13 0 &gpio0 9 0>,	/* D7 */
+			   <14 0 &gpio0 14 0>,	/* D8 */
+			   <15 0 &gpio0 24 0>,	/* D9 */
+			   <16 0 &gpio0 22 0>,	/* D10 */
+			   <17 0 &gpio0 23 0>,	/* D11 */
+			   <18 0 &gpio0 18 0>,	/* D12 */
+			   <19 0 &gpio0 20 0>,	/* D13 */
+			   <20 0 &gpio0 15 0>,	/* D14 */
+			   <21 0 &gpio0 16 0>;	/* D15 */
+	};
+
+	arduino_adc: analog-connector {
+		compatible = "arduino,uno-adc";
+		#io-channel-cells = <1>;
+		io-channel-map = <0 &adc 1>,	/* A0 = P0.4 = AIN1 */
+				 <1 &adc 2>,	/* A1 = P0.5 = AIN2 */
+				 <2 &adc 4>,	/* A2 = P0.28 = AIN4 */
+				 <3 &adc 5>,	/* A3 = P0.29 = AIN5 */
+				 <4 &adc 6>,	/* A4 = P0.30 = AIN6 */
+				 <5 &adc 7>;	/* A5 = P0.31 = AIN7 */
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		pwm-led0 = &pwm_led0;
+		pwm-led1 = &pwm_led1;
+		pwm-led2 = &pwm_led2;
```

Comment:
You could use here:
```
		pwm-led0 = &red_pwm_led;
		pwm-led1 = &green_pwm_led;
		pwm-led2 = &blue_pwm_led;
```
and avoid defining `pwm_led0`, `pwm_led1`, and `pwm_led2` nodes.